backend/job/query.py

```diff
@@ -30,7 +30,9 @@ def create_job(payload: dict) -> Job:
         description=payload["description"] if payload.get("description") else "",
         notes=payload["notes"] if payload.get("notes") else "",
         cover_letter=payload["cover_letter"] if payload.get("cover_letter") else "",
-        deadlines=payload["deadlines"] if payload.get("deadlines") else None,
+        deadlines=payload["deadlines"]
+        if payload.get("deadlines")
+        else None,  # I want an empty json.dumps([]) here but causes issues
```

Comment:
I don't think this `else` is ever actually reached unless there's an error so I don't think it really matters what value we use - since its a `JSONField` we could just use an empty string